app/src/main/java/com/jerboa/db/AppDBMigrations.kt

```diff
@@ -261,6 +261,31 @@ val MIGRATION_19_20 = object : Migration(19, 20) {
     }
 }
 
+val MIGRATION_20_21 = object : Migration(20, 21) {
+    override fun migrate(database: SupportSQLiteDatabase) {
+        database.execSQL(UPDATE_APP_CHANGELOG_UNVIEWED)
+        database.execSQL(
+            "ALTER TABLE AppSettings ADD COLUMN save_search_history INTEGER NOT NULL DEFAULT 1",
+        )
+        database.execSQL(
+            """
+                CREATE TABLE IF NOT EXISTS SearchHistory(
+                    id INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL, 
+                    account_id INTEGER,
```

Comment:
account_id should also be not null, no? And it'd probably be clearer to move the references / fk relation into this line.